src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -2366,6 +2371,55 @@ public override BoundNode VisitConditionalOperator(BoundConditionalOperator node
             return null;
         }
 
+        private BoundNode VisitRefConditionalOperator(BoundConditionalOperator node)
```

Comment:
ah, I see. This way it might work.

---
In reply to: [105692349](https://github.com/dotnet/roslyn/pull/17436#discussion_r105692349) [](ancestors = 105692349)